LayoutTests/platform/mac/TestExpectations

```diff
@@ -1743,13 +1743,13 @@ inspector/unit-tests/test-harness-expect-functions.html [ Pass Crash ]
 editing/selection/select-bidi-run.html [ Pass Timeout ]
 
 # <rdar://problem/60873976> [ Mac Guard Malloc ] imported/w3c/web-platform-tests/WebCryptoAPI/generateKey/failures_AES-CBC.https.any.worker.html is flaky crashing.
-imported/w3c/web-platform-tests/WebCryptoAPI/generateKey/failures_AES-CBC.https.any.worker.html [ Pass Crash ]
+[ Guard-Malloc ] imported/w3c/web-platform-tests/WebCryptoAPI/generateKey/failures_AES-CBC.https.any.worker.html [ Pass Crash ]
 
 # In-App Browser Privacy requires functions that are currently in WebKitAdditions.
 http/tests/in-app-browser-privacy/ [ Skip ]
 
 # <rdar://problem/61264058> [ iOS and Mac with Guard-Malloc and wk2 or Stress-GC ] imported/w3c/web-platform-tests/css/css-sizing/percentage-height-in-flexbox.html is flaky failing.
-imported/w3c/web-platform-tests/css/css-sizing/percentage-height-in-flexbox.html [ Pass Failure ]
+[ Guard-Malloc ] imported/w3c/web-platform-tests/css/css-sizing/percentage-height-in-flexbox.html [ Pass Failure ]
```

Comment:
Same thing - what is GuardMalloc specific here?

I think that we should only use this expectation for tests that are somehow invalid underGuardMalloc (the only case I know of is allocating too much memory).